src/index.d.ts

```diff
@@ -0,0 +1,72 @@
+/// <reference types="react" />
+
+import React from "react";
```

Comment:
Must be:
`import * as React from "react";`

React got no default export...

![2018-04-10_15h22_53](https://user-images.githubusercontent.com/7689458/38559892-86fcd93e-3cd4-11e8-99e5-eeb1bd780ea1.png)

Reply:
Thanks @Zanaxhir . It has been fixed by #36 . 
By the way, my vscode does not complain this ts issue, Do you know why? Or how do you see this issue?